client.go

```diff
@@ -66,6 +66,16 @@ func (c *Client) Post(resource string, data url.Values) ([]byte, error) {
 	return c.do(req)
 }
 
+func (c *Client) Put(resource string, data url.Values) ([]byte, error) {
+	req, err := http.NewRequest("PUT", c.endpoint+resource, strings.NewReader(data.Encode()))
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
```

Comment:
If you want to you can take a look at the breaking changes here: https://github.com/Luzifer/go-trello/compare/master...Luzifer:dev - switching from `[]byte` responses to objects also will be breaking for everyone relying on getting `[]byte`…